PurchasesTests/Networking/HTTPClientTests.swift

```diff
@@ -834,3 +812,20 @@ extension HTTPClientTests {
         expect(completionCalled).toEventually(equal(true), timeout: .seconds(1))
     }
 }
+
+private extension HTTPClientTests {
+
+    func extractRequestNumber(from urlRequest: URLRequest) -> Int {
+        do {
+            let requestData = urlRequest.ohhttpStubs_httpBody!
+            let requestBodyDict = try XCTUnwrap(try JSONSerialization.jsonObject(with: requestData,
+                                                                                 options: []) as? [String: Any])
+            return try XCTUnwrap(requestBodyDict["requestNumber"] as? Int)
+        } catch {
+            XCTFail(error.localizedDescription)
+        }
+
+        return -999
```

Comment:
This return will never be executed

Reply:
Feel free to do this in another PR, but to document that I would replace it with `fatalError("Unreachable")`

Reply:
Sure! Much better 👍🏼